### ISSUE TITLE
Question regarding pthreads being enabled...

### ISSUE THREAD

**Issue by newcapricasean** [rdp/ffmpeg-windows-build-helpers#680]
This isn't so much of an issue, as it is a feature inquiry and request. I'm still new to this linux, compiling, and FFMPEG stuff... BUT, I want to learn it... So, when I look at the options included in the output of my ffmpeg executables, the only "threads" mentioned is "--disable-w32threads". I don't see any mention of "pthreads" or "winpthreads" being enabled, while encoding audio with the Fraunhofer AAC codec. While a novice at hacking compile scripts, I don't seem to see a mention of it being "--enabled", except for under the build_mplayer section. Does this mean that the FFMPEG is not set up to utilize multiple threads effectively? I'm primarily interested in the Fraunhofer AAC encoder, from FFMPEG. winpthreads seems to have lost support, but I did find pthreads4w, online, and downloaded source code for it, which was from 2018... seemingly much more recent than the other options... Below is the web link... Is this possible, or even necessary? Does the FFMPEG executable already utilize numerous threads as best it can, with the current scripted build? I have an AMD Ryzen Threadripper 3990x, which has 64 cores, and 128 multithreading, on Windows 11, with WSL, and Ubuntu 22.04 LTS. I will also include my current command line that I am using below. As you can see, I'm already customizing it with the -march and -mtune parameters for my processor type. By the way, that customization does make a rather huge difference! It's somewhere in the neighborhood of a 35% speed increase, just by adding the -march and -mtune parameters!!! Please let me know whether or not pthreads would help, or whether it is already ideally customized, already... Thanks, in advance!

https://sourceforge.net/projects/pthreads4w/

>>>
./cross_compile_ffmpeg.sh --gcc-cpu-count=64 --disable-nonfree=n --build-intel-qsv=n --compiler-flavors=win64 --enable-gpl=y --build-lsw=y --prefer-stable=y --cflags="-march=znver2 -O2 -mtune=znver2 -O2 -pipe"
<<<

**Comment by newcapricasean** [comment]
Additionally, when I compared the x265 cli encoder I downloaded from http://msystem.waw.pl/x265/, with the built in one with the FFMPEG I compiled, the x265 one I'd downloaded was much faster, and that was with the settings being the same, through the CLI. Even the output, from the x265 cli, showed identical configuration settings and CPU optimizations used. The resulting encoded files were similarly the exact same size.
